src/main/java/com/assignment/Assignment9.java

```diff
@@ -0,0 +1,18 @@
+package com.assignment;
+
+import java.util.ArrayList;
+
+public class Assignment9 {
+
+    public ArrayList getEvenNumber(int min, int max) {
+        int index = min;
```

Comment:
No need of index directly use, use directly min value below, and after that change the name from min to number or something else, bcoz we are going to increment that number